### ISSUE TITLE
Half hours

### ISSUE THREAD

**Issue by matthijs-neijenhuijs** [mcuznz/jquery.dayparts#4]
It will be great if its possible to use for example 17:30.

**Comment by etorrano** [comment]
I agree with @isomis. I need to add the function to select at least half hours, and in the best case the exact minutes, for example 8:15 Hs. If you have any idea or suggestion to implement it tell me because maybe I have to do it.
